docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE*Tattle
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following issues.
+
+.Cannot create a Jira instance behind a proxy
```

Comment:
This is listed as both a known issue and a resolved bug. Remove from here

Reply:
Done